src/main/java/org/linkeddatafragments/solver/ReorderTransformationLDF.java

```diff
@@ -61,7 +61,6 @@ private void initializeMatcher() {
         matcher.addPattern(new Pattern(numTriples, VAR, VAR, VAR));     // ???
     }
 
-    @Override
```

Comment:
In the same way, it seems. [Only the class name was changed.](https://github.com/apache/jena/commit/31a36d32d238ca803097f6769367745ed06c7b51)